config.sh

```diff
@@ -24,7 +24,7 @@ function pre_build {
         export CFLAGS="-I/usr/local/include $ADD_CFLAGS $CFLAGS"
         export LDFLAGS="-L/usr/local/lib $ARCHFLAGS"
         echo "Downloading GMP"
-        curl -O https://gmplib.org/download/gmp/gmp-$GMP_VERSION.tar.lz
+        curl -O https://ftp.gnu.org/gnu/gmp/gmp-${GMP_VERSION}.tar.lz
```

Comment:
Is there a direct FTP link, too?

```suggestion
        curl -O ftp://ftp.gnu.org/gnu/gmp/gmp-${GMP_VERSION}.tar.lz
```